olcUTIL_Geometry2D.h

```diff
@@ -2457,7 +2457,8 @@ namespace olc::utils::geom2d
 			if (s2 < 0)
 				return { q.origin + q.direction * s1 };
 
-			return { q.origin + q.direction * std::min(s1, s2) };
+			const auto minmax = std::minmax(s1, s2);

```

Comment:
I would recommend using structured bindings here to facilitate a better name than just the function name itself to improve readability:

```cpp
const auto& [first_intersection_distance, second_intersection_distance] = std::minmax(s1, s2);
return {q.origin + q.direction * first_intersection_distance, q.origin + q.direction * second_intersection_distance};
//...
```

Reply:
Good point! At my day job I am responsible for the PR reviews and I would've commented the exact same thing, so I don't know why I didn't do it in the first place :D

Reply:
Done. Left out the reference, though, since we're dealing with fundamental types.